statsd/install_statsd.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+rm -rf ~/build_statsd
```

Comment:
Maybe we could do this work in `/tmp/` since we don't need it to hang around afterwards. (and related... i can't believe there's not an official deb/PPA!)